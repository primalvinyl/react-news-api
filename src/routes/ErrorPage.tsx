import React from 'react';
import styles from './ErrorPage.module.scss';

const ErrorPage = (): React.ReactElement => {
    return (
        <div className={styles.root}>
            <div className={styles.wrapper}>
                <h1>404</h1>
            </div>
        </div>
    );
};

export default ErrorPage;
