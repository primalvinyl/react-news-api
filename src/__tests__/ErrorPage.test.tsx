import React from 'react';
import { shallow } from 'enzyme';
import Renderer from 'react-test-renderer';
import ErrorPage from '../routes/ErrorPage';

describe('ErrorPage', () => {
    it('renders', () => {
        const myWrapper = shallow(<ErrorPage />);
        expect(myWrapper.exists());
    });
    it('matches snapshot', () => {
        const component = Renderer.create(<ErrorPage />);
        expect(component.toJSON()).toMatchSnapshot();
    });
});
