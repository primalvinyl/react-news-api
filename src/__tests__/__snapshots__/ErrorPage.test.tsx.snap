// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorPage matches snapshot 1`] = `
<div
  className="root"
>
  <div
    className="wrapper"
  >
    <h1>
      404
    </h1>
  </div>
</div>
`;
